can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -98,18 +99,14 @@ def __init__(self, channel, can_filters=None, **config):
         logger.info("CAN Filters: {}".format(can_filters))
         logger.info("Got configuration of: {}".format(config))
 
-        self._use_system_timestamp = bool(
-            config.get('use_system_timestamp', False)
-        )
-        try:
-            channel = int(channel)
-        except ValueError:
-            channel = getattr(ics, "NETID_{}".format(channel.upper()), -1)
-            if channel == -1:
-                raise ValueError(
-                    'channel must be an integer or '
-                    'a valid ICS channel name'
-                )
+        if isinstance(channel, (list, tuple)):
```

Comment:
I don't know whether it is is helpful to support the string type here. I would propose that we only support integers and iterables of such. That would not be harder to use (if not even simpler to read in the user code) and we could check the types differently:

```Python
from collections import Iterable

[...]

try:
   # assume int
   self.channels = [int(channels)]
except (ValueError, TypeError):
   # assume iterable of int
   self.channels = [NeoViBus.channel_to_netid(ch) for ch in channels]
```

This would allow types of all iterables that iterate over integers or over objects that can be converted to integers.

Reply:
Strings must always be supported or it won’t work with our scripts or configuration files.

Reply:
Okay, I see.